{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Label Functions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "from cord import ResearchPapers\n",
    "from cord.core import describe_dataframe, find_data_dir, SARS_COV_2_DATE\n",
    "from pathlib import Path, PurePath\n",
    "import pandas as pd\n",
    "pd.options.display.max_colwidth = 2000\n",
    "pd.options.display.max_rows = 200"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading metadata from data\\CORD-19-research-challenge\n",
      "Cleaning metadata\n",
      "Applying tags to metadata\n",
      "\n",
      "Indexing research papers\n",
      "Creating the BM25 index from the abstracts of the papers\n",
      "Use index=\"text\" if you want to index the texts of the paper instead\n",
      "Finished Indexing in 40.0 seconds\n"
     ]
    }
   ],
   "source": [
    "research_papers = ResearchPapers.load()\n",
    "metadata = research_papers.metadata\n",
    "#metadata = pd.read_csv(PurePath(find_data_dir()) / 'metadata.csv')\n",
    "#describe_dataframe(metadata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata = research_papers.metadata.drop(columns=['index_tokens'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "covid_papers = research_papers.covid_related()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>journal</th>\n",
       "      <th>authors</th>\n",
       "      <th>published</th>\n",
       "      <th>when</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>A mathematical model for simulating the transmission of Wuhan novel Coronavirus</td>\n",
       "      <td>As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R0) was calculated from the RP model to assess the transmissibility of the 2019-nCoV.</td>\n",
       "      <td></td>\n",
       "      <td>Tianmu Chen; Jia Rui; Qiupeng Wang; Zeyu Zhao; Jing-An Cui; Ling Yin</td>\n",
       "      <td>2020-01-19</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China</td>\n",
       "      <td>Detailed genomic and structure-based analysis of a new coronavirus, namely 2019-nCoV, showed that the new virus is a new type of bat coronavirus and is genetically fairly distant from the human SARS coronavirus. Structure analysis of the spike (S) protein of this new virus showed that its S protein only binds weakly to the ACE2 receptor on human cells whereas the human SARS coronavirus exhibits strongly affinity to the ACE receptor. These findings suggest that the new virus does not readily transmit between humans and should theoretically not able to cause very serious human infection. These data are important to guide design of infection control policy and inform the public on the nature of threat imposed by 2019-nCov when results of direct laboratory tests on this virus are not expected to be available in the near future.</td>\n",
       "      <td></td>\n",
       "      <td>Ning Dong; Xuemei Yang; Lianwei Ye; Kaichao Chen; Edward Wai-Chi Chan; Mengsu Yang; Sheng Chen</td>\n",
       "      <td>2020-01-21</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm</td>\n",
       "      <td>The recent outbreak of pneumonia in Wuhan, China caused by the 2019 Novel Coronavirus (2019-nCoV) emphasizes the importance of detecting novel viruses and predicting their risks of infecting people. In this report, we introduced the VHP (Virus Host Prediction) to predict the potential hosts of viruses using deep learning algorithm. Our prediction suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially the severe acute respiratory syndrome coronavirus (SARS-CoV), Bat SARS-like Coronaviruses and the Middle East respiratory syndrome coronavirus (MERS-CoV). Based on our prediction, compared to the Coronaviruses infecting other vertebrates, bat coronaviruses are assigned with more similar infectivity patterns with 2019-nCoVs. Furthermore, by comparing the infectivity patterns of all viruses hosted on vertebrates, we found mink viruses show a closer infectivity pattern to 2019-nCov. These consequences of infectivity pattern analysis illustrate that bat and mink may be two candidate reservoirs of 2019-nCov.These results warn us to beware of 2019-nCoV and guide us to further explore the properties and reservoir of it.One Sentence It is of great value to identify whether a newly discovered virus has the risk of infecting human. Guo et al. proposed a virus host prediction method based on deep learning to detect what kind of host a virus can infect with DNA sequence as input. Applied to the Wuhan 2019 Novel Coronavirus, our prediction demonstrated that several vertebrate-infectious coronaviruses have strong potential to infect human. This method will be helpful in future viral analysis and early prevention and control of viral pathogens.</td>\n",
       "      <td></td>\n",
       "      <td>Qian Guo; Mo Li; Chunhui Wang; Peihong Wang; Zhencheng Fang; Jie tan; Shufang Wu; Yonghong Xiao; Huaiqiu Zhu</td>\n",
       "      <td>2020-01-24</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin</td>\n",
       "      <td>Since the SARS outbreak 18 years ago, a large number of severe acute respiratory syndrome related coronaviruses (SARSr-CoV) have been discovered in their natural reservoir host, bats1-4. Previous studies indicated that some of those bat SARSr-CoVs have the potential to infect humans5-7. Here we report the identification and characterization of a novel coronavirus (nCoV-2019) which caused an epidemic of acute respiratory syndrome in humans, in Wuhan, China. The epidemic, started from December 12th, 2019, has caused 198 laboratory confirmed infections with three fatal cases by January 20th, 2020. Full-length genome sequences were obtained from five patients at the early stage of the outbreak. They are almost identical to each other and share 79.5% sequence identify to SARS-CoV. Furthermore, it was found that nCoV-2019 is 96% identical at the whole genome level to a bat coronavirus. The pairwise protein sequence analysis of seven conserved non-structural proteins show that this virus belongs to the species of SARSr-CoV. The nCoV-2019 virus was then isolated from the bronchoalveolar lavage fluid of a critically ill patient, which can be neutralized by sera from several patients. Importantly, we have confirmed that this novel CoV uses the same cell entry receptor, ACE2, as SARS-CoV.</td>\n",
       "      <td></td>\n",
       "      <td>Peng Zhou; Xing-Lou Yang; Xian-Guang Wang; Ben Hu; Lei Zhang; Wei Zhang; Hao-Rui Si; Yan Zhu; Bei Li; Chao-Lin Huang; Hui-Dong Chen; Jing Chen; Yun Luo; Hua Guo; Ren-Di Jiang; Mei-Qin Liu; Ying Chen; Xu-Rui Shen; Xi Wang; Xiao-Shuang Zheng; Kai Zhao; Quan-Jiao Chen; Fei Deng; Lin-Lin Liu; Bing Yan; Fa-Xian Zhan; Yan-Yi Wang; Geng-Fu Xiao; Zheng-Li Shi</td>\n",
       "      <td>2020-01-23</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</td>\n",
       "      <td>Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.</td>\n",
       "      <td></td>\n",
       "      <td>Michael Letko; Vincent Munster</td>\n",
       "      <td>2020-01-22</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45746</th>\n",
       "      <td>Comments on \"Preliminary estimation of the basic reproduction number of novel Coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven Analysis in the early phase of the outbreak\"</td>\n",
       "      <td>Comments on \"Preliminary estimation of the basic reproduction number of novel Coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven Analysis in the early phase of the outbreak\"</td>\n",
       "      <td>International Journal of Infectious Diseases</td>\n",
       "      <td>Dhungana, Hom Nath</td>\n",
       "      <td>2020-02-20</td>\n",
       "      <td>1 month ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45747</th>\n",
       "      <td>Quantitative computed tomography analysis for stratifying the severity of Coronavirus Disease 2019</td>\n",
       "      <td>Purpose To examine the feasibility of using a computer tool for stratifying the severity of Coronavirus Disease 2019 (COVID-19) based on computed tomography (CT) images. Materials and methods We retrospectively examined 44 confirmed COVID-19 cases. All cases were evaluated separately by radiologists (visually) and through an in-house computer software. The degree of lesions was visually scored by the radiologist, as follows, for each of the 5 lung lobes: 0, no lesion present; 1, &lt;1/3 involvement; 2, &gt;1/3 and &lt; 2/3 involvement; and 3, &gt;2/3 involvement. Lesion density was assessed based on the proportion of ground-glass opacity (GGO), consolidation and fibrosis of the lesions. The parameters obtained using the computer tool included lung volume (mL), lesion volume (mL), lesion percentage (%), and mean lesion density (HU) of the whole lung, right lung, left lung, and each lobe. The scores obtained by the radiologists and quantitative results generated by the computer software were tested for correlation. A Chi-square test was used to test the consistency of radiologist- and computer-derived lesion percentage in the right/left lung, upper/lower lobe, and each of the 5 lobes. Result The results showed a strong to moderate correlation between lesion percentage scores obtained by radiologists and the computer software (r ranged from 0.7679 to 0.8373, P &lt; 0.05), and a moderate correlation between the proportion of GGO and mean lesion density (r = −0.5894, P &lt; 0.05), and proportion of consolidation and mean lesion density (r = 0.6282, P &lt; 0.05). Computer-aided quantification showed a statistical significant higher lesion percentage for lower lobes than that assessed by the radiologists (χ2 = 8.160, P = 0.004). Conclusions Our experiments demonstrated that the computer tool could reliably and accurately assess the severity and distribution of pneumonia on CT scans.</td>\n",
       "      <td>Journal of Pharmaceutical Analysis</td>\n",
       "      <td>Shen, Cong; Yu, Nan; Cai, Shubo; Zhou, Jie; Sheng, Jiexin; Liu, Kang; Zhou, Heping; Guo, Youmin; Niu, Gang</td>\n",
       "      <td>2020-03-06</td>\n",
       "      <td>4 weeks ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45753</th>\n",
       "      <td>The SARS-CoV-2 outbreak from a one health perspective</td>\n",
       "      <td>The SARS-CoV-2 is a new human coronavirus candidate recently detected in China that is now reported in people on inhabited continents. The virus shares a high level of identity with some bat coronaviruses and is recognised as a potentially zoonotic virus. We are utilizing the One Health concept to understand the emergence of the virus, as well as to point to some possible control strategies that might reduce the spread of the virus across the globe; thus, containment of such virus would be possible.</td>\n",
       "      <td>One Health</td>\n",
       "      <td>Hemida, Maged Gomaa; Ba Abduallah, Mohammed M.</td>\n",
       "      <td>2020-03-16</td>\n",
       "      <td>2 weeks ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45755</th>\n",
       "      <td>Histopathologic Changes and SARS–CoV-2 Immunostaining in the Lung of a Patient With COVID-19</td>\n",
       "      <td>Histopathologic Changes and SARS–CoV-2 Immunostaining in the Lung of a Patient With COVID-19</td>\n",
       "      <td>Ann Intern Med</td>\n",
       "      <td>Zhang, Huilan; Zhou, Peng; Wei, Yanqiu; Yue, Huihui; Wang, Yi; Hu, Ming; Zhang, Shu; Cao, Tanze; Yang, Chengqing; Li, Ming; Guo, Guangyun; Chen, Xianxiang; Chen, Ying; Lei, Mei; Liu, Huiguo; Zhao, Jianping; Peng, Peng; Wang, Cong-Yi; Du, Ronghui</td>\n",
       "      <td>2020-03-12</td>\n",
       "      <td>3 weeks ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45771</th>\n",
       "      <td>Why the WHO won't use the p-word</td>\n",
       "      <td>There are no criteria for a pandemic, but covid-19 looks like one, says Debora MacKenzie</td>\n",
       "      <td>New Scientist</td>\n",
       "      <td>MacKenzie, Debora</td>\n",
       "      <td>2020-03-07</td>\n",
       "      <td>4 weeks ago</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2427 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "<cord.cord19.ResearchPapers at 0x1580bd00eb8>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "covid_papers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Wall time: 5.63 s\n"
     ]
    }
   ],
   "source": [
    "COVID_TERMS = ['covid', 'sars-?n?cov-?2','2019-ncov', 'novel coronavirus', 'sars coronavirus 2']\n",
    "COVID_SEARCH = f\".*({'|'.join(COVID_TERMS)})\"\n",
    "NOVEL_CORONAVIRUS = '.*novel coronavirus'\n",
    "WUHAN_OUTBREAK = 'wuhan'\n",
    "    \n",
    "def tag_covid(data):  \n",
    "    abstract = data.abstract.fillna('')\n",
    "    since_covid = (data.published > SARS_COV_2_DATE) | (data.published.isnull())\n",
    "    covid_term_match = since_covid & abstract.str.match(COVID_SEARCH, case=False)\n",
    "    wuhan_outbreak = since_covid & abstract.str.match('.*(wuhan|hubei)', case=False)\n",
    "    covid_match = covid_term_match | wuhan_outbreak\n",
    "    data['is_covid'] = False\n",
    "    data.loc[covid_match, 'is_covid'] = True\n",
    "    return data\n",
    "    \n",
    "%time metadata = metadata.pipe(tag_covid)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>is_covid</th>\n",
       "      <th>covid_related</th>\n",
       "      <th>abstract</th>\n",
       "      <th>published</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [is_covid, covid_related, abstract, published]\n",
       "Index: []"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "COLS = ['is_covid', 'covid_related', 'abstract', 'published']\n",
    "mismatches = metadata[metadata.is_covid != metadata.covid_related]\n",
    "\n",
    "mismatches =mismatches.loc[(mismatches.published < SARS_COV_2_DATE) &\n",
    "                          ~(metadata.is_covid), COLS].sort_values('published', ascending=False)\n",
    "display(len(mismatches))\n",
    "mismatches"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'We report co-infection with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and influenza A virus in a patient with pneumonia in China. The case highlights possible co-detection of known respiratory viruses. We noted low sensitivity of upper respiratory specimens for SARS-CoV-2, which could further complicate recognition of the full extent of disease.'"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "'.*(covid|sars-?n?cov-?2|2019-ncov|novel coronavirus|sars coronavirus 2)'"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<re.Match object; span=(0, 291), match='We report co-infection with severe acute respirat>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import re\n",
    "paper = research_papers[1853]\n",
    "display(paper.abstract)\n",
    "display(COVID_SEARCH)\n",
    "re.match(COVID_SEARCH, paper.abstract, re.IGNORECASE)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'.*(covid|sars-?n?cov-?2|2019-ncov|novel coronavirus|sars coronavirus 2)'"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "COVID_SEARCH"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading metadata from data\\CORD-19-research-challenge\n",
      "Cleaning metadata\n",
      "Applying tags to metadata\n",
      "\n",
      "Indexing research papers\n",
      "Creating the BM25 index from the abstracts of the papers\n",
      "Use index=\"text\" if you want to index the texts of the paper instead\n",
      "Finished Indexing in 38.0 seconds\n"
     ]
    }
   ],
   "source": [
    "research_papers = ResearchPapers.load()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "covid_papers = research_papers.covid_related()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2427"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(covid_papers)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>journal</th>\n",
       "      <th>authors</th>\n",
       "      <th>published</th>\n",
       "      <th>when</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>A mathematical model for simulating the transmission of Wuhan novel Coronavirus</td>\n",
       "      <td>As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probable be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from a seafood market (reservoir) to people, we simplified the model as Reservoir-People transmission network model. The basic reproduction number (R0) was calculated from the RP model to assess the transmissibility of the 2019-nCoV.</td>\n",
       "      <td></td>\n",
       "      <td>Tianmu Chen; Jia Rui; Qiupeng Wang; Zeyu Zhao; Jing-An Cui; Ling Yin</td>\n",
       "      <td>2020-01-19</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China</td>\n",
       "      <td>Detailed genomic and structure-based analysis of a new coronavirus, namely 2019-nCoV, showed that the new virus is a new type of bat coronavirus and is genetically fairly distant from the human SARS coronavirus. Structure analysis of the spike (S) protein of this new virus showed that its S protein only binds weakly to the ACE2 receptor on human cells whereas the human SARS coronavirus exhibits strongly affinity to the ACE receptor. These findings suggest that the new virus does not readily transmit between humans and should theoretically not able to cause very serious human infection. These data are important to guide design of infection control policy and inform the public on the nature of threat imposed by 2019-nCov when results of direct laboratory tests on this virus are not expected to be available in the near future.</td>\n",
       "      <td></td>\n",
       "      <td>Ning Dong; Xuemei Yang; Lianwei Ye; Kaichao Chen; Edward Wai-Chi Chan; Mengsu Yang; Sheng Chen</td>\n",
       "      <td>2020-01-21</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm</td>\n",
       "      <td>The recent outbreak of pneumonia in Wuhan, China caused by the 2019 Novel Coronavirus (2019-nCoV) emphasizes the importance of detecting novel viruses and predicting their risks of infecting people. In this report, we introduced the VHP (Virus Host Prediction) to predict the potential hosts of viruses using deep learning algorithm. Our prediction suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially the severe acute respiratory syndrome coronavirus (SARS-CoV), Bat SARS-like Coronaviruses and the Middle East respiratory syndrome coronavirus (MERS-CoV). Based on our prediction, compared to the Coronaviruses infecting other vertebrates, bat coronaviruses are assigned with more similar infectivity patterns with 2019-nCoVs. Furthermore, by comparing the infectivity patterns of all viruses hosted on vertebrates, we found mink viruses show a closer infectivity pattern to 2019-nCov. These consequences of infectivity pattern analysis illustrate that bat and mink may be two candidate reservoirs of 2019-nCov.These results warn us to beware of 2019-nCoV and guide us to further explore the properties and reservoir of it.One Sentence It is of great value to identify whether a newly discovered virus has the risk of infecting human. Guo et al. proposed a virus host prediction method based on deep learning to detect what kind of host a virus can infect with DNA sequence as input. Applied to the Wuhan 2019 Novel Coronavirus, our prediction demonstrated that several vertebrate-infectious coronaviruses have strong potential to infect human. This method will be helpful in future viral analysis and early prevention and control of viral pathogens.</td>\n",
       "      <td></td>\n",
       "      <td>Qian Guo; Mo Li; Chunhui Wang; Peihong Wang; Zhencheng Fang; Jie tan; Shufang Wu; Yonghong Xiao; Huaiqiu Zhu</td>\n",
       "      <td>2020-01-24</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin</td>\n",
       "      <td>Since the SARS outbreak 18 years ago, a large number of severe acute respiratory syndrome related coronaviruses (SARSr-CoV) have been discovered in their natural reservoir host, bats1-4. Previous studies indicated that some of those bat SARSr-CoVs have the potential to infect humans5-7. Here we report the identification and characterization of a novel coronavirus (nCoV-2019) which caused an epidemic of acute respiratory syndrome in humans, in Wuhan, China. The epidemic, started from December 12th, 2019, has caused 198 laboratory confirmed infections with three fatal cases by January 20th, 2020. Full-length genome sequences were obtained from five patients at the early stage of the outbreak. They are almost identical to each other and share 79.5% sequence identify to SARS-CoV. Furthermore, it was found that nCoV-2019 is 96% identical at the whole genome level to a bat coronavirus. The pairwise protein sequence analysis of seven conserved non-structural proteins show that this virus belongs to the species of SARSr-CoV. The nCoV-2019 virus was then isolated from the bronchoalveolar lavage fluid of a critically ill patient, which can be neutralized by sera from several patients. Importantly, we have confirmed that this novel CoV uses the same cell entry receptor, ACE2, as SARS-CoV.</td>\n",
       "      <td></td>\n",
       "      <td>Peng Zhou; Xing-Lou Yang; Xian-Guang Wang; Ben Hu; Lei Zhang; Wei Zhang; Hao-Rui Si; Yan Zhu; Bei Li; Chao-Lin Huang; Hui-Dong Chen; Jing Chen; Yun Luo; Hua Guo; Ren-Di Jiang; Mei-Qin Liu; Ying Chen; Xu-Rui Shen; Xi Wang; Xiao-Shuang Zheng; Kai Zhao; Quan-Jiao Chen; Fei Deng; Lin-Lin Liu; Bing Yan; Fa-Xian Zhan; Yan-Yi Wang; Geng-Fu Xiao; Zheng-Li Shi</td>\n",
       "      <td>2020-01-23</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</td>\n",
       "      <td>Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.</td>\n",
       "      <td></td>\n",
       "      <td>Michael Letko; Vincent Munster</td>\n",
       "      <td>2020-01-22</td>\n",
       "      <td>2 months ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45746</th>\n",
       "      <td>Comments on \"Preliminary estimation of the basic reproduction number of novel Coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven Analysis in the early phase of the outbreak\"</td>\n",
       "      <td>Comments on \"Preliminary estimation of the basic reproduction number of novel Coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven Analysis in the early phase of the outbreak\"</td>\n",
       "      <td>International Journal of Infectious Diseases</td>\n",
       "      <td>Dhungana, Hom Nath</td>\n",
       "      <td>2020-02-20</td>\n",
       "      <td>1 month ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45747</th>\n",
       "      <td>Quantitative computed tomography analysis for stratifying the severity of Coronavirus Disease 2019</td>\n",
       "      <td>Purpose To examine the feasibility of using a computer tool for stratifying the severity of Coronavirus Disease 2019 (COVID-19) based on computed tomography (CT) images. Materials and methods We retrospectively examined 44 confirmed COVID-19 cases. All cases were evaluated separately by radiologists (visually) and through an in-house computer software. The degree of lesions was visually scored by the radiologist, as follows, for each of the 5 lung lobes: 0, no lesion present; 1, &lt;1/3 involvement; 2, &gt;1/3 and &lt; 2/3 involvement; and 3, &gt;2/3 involvement. Lesion density was assessed based on the proportion of ground-glass opacity (GGO), consolidation and fibrosis of the lesions. The parameters obtained using the computer tool included lung volume (mL), lesion volume (mL), lesion percentage (%), and mean lesion density (HU) of the whole lung, right lung, left lung, and each lobe. The scores obtained by the radiologists and quantitative results generated by the computer software were tested for correlation. A Chi-square test was used to test the consistency of radiologist- and computer-derived lesion percentage in the right/left lung, upper/lower lobe, and each of the 5 lobes. Result The results showed a strong to moderate correlation between lesion percentage scores obtained by radiologists and the computer software (r ranged from 0.7679 to 0.8373, P &lt; 0.05), and a moderate correlation between the proportion of GGO and mean lesion density (r = −0.5894, P &lt; 0.05), and proportion of consolidation and mean lesion density (r = 0.6282, P &lt; 0.05). Computer-aided quantification showed a statistical significant higher lesion percentage for lower lobes than that assessed by the radiologists (χ2 = 8.160, P = 0.004). Conclusions Our experiments demonstrated that the computer tool could reliably and accurately assess the severity and distribution of pneumonia on CT scans.</td>\n",
       "      <td>Journal of Pharmaceutical Analysis</td>\n",
       "      <td>Shen, Cong; Yu, Nan; Cai, Shubo; Zhou, Jie; Sheng, Jiexin; Liu, Kang; Zhou, Heping; Guo, Youmin; Niu, Gang</td>\n",
       "      <td>2020-03-06</td>\n",
       "      <td>4 weeks ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45753</th>\n",
       "      <td>The SARS-CoV-2 outbreak from a one health perspective</td>\n",
       "      <td>The SARS-CoV-2 is a new human coronavirus candidate recently detected in China that is now reported in people on inhabited continents. The virus shares a high level of identity with some bat coronaviruses and is recognised as a potentially zoonotic virus. We are utilizing the One Health concept to understand the emergence of the virus, as well as to point to some possible control strategies that might reduce the spread of the virus across the globe; thus, containment of such virus would be possible.</td>\n",
       "      <td>One Health</td>\n",
       "      <td>Hemida, Maged Gomaa; Ba Abduallah, Mohammed M.</td>\n",
       "      <td>2020-03-16</td>\n",
       "      <td>2 weeks ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45755</th>\n",
       "      <td>Histopathologic Changes and SARS–CoV-2 Immunostaining in the Lung of a Patient With COVID-19</td>\n",
       "      <td>Histopathologic Changes and SARS–CoV-2 Immunostaining in the Lung of a Patient With COVID-19</td>\n",
       "      <td>Ann Intern Med</td>\n",
       "      <td>Zhang, Huilan; Zhou, Peng; Wei, Yanqiu; Yue, Huihui; Wang, Yi; Hu, Ming; Zhang, Shu; Cao, Tanze; Yang, Chengqing; Li, Ming; Guo, Guangyun; Chen, Xianxiang; Chen, Ying; Lei, Mei; Liu, Huiguo; Zhao, Jianping; Peng, Peng; Wang, Cong-Yi; Du, Ronghui</td>\n",
       "      <td>2020-03-12</td>\n",
       "      <td>3 weeks ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45771</th>\n",
       "      <td>Why the WHO won't use the p-word</td>\n",
       "      <td>There are no criteria for a pandemic, but covid-19 looks like one, says Debora MacKenzie</td>\n",
       "      <td>New Scientist</td>\n",
       "      <td>MacKenzie, Debora</td>\n",
       "      <td>2020-03-07</td>\n",
       "      <td>4 weeks ago</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2427 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "<cord.cord19.ResearchPapers at 0x15830818a58>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "covid_papers"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## SARS Labeling"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading metadata from data\\CORD-19-research-challenge\n",
      "Cleaning metadata\n",
      "Applying tags to metadata\n",
      "\n",
      "Indexing research papers\n",
      "Creating the BM25 index from the abstracts of the papers\n",
      "Use index=\"text\" if you want to index the texts of the paper instead\n",
      "Finished Indexing in 45.0 seconds\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>abstract</th>\n",
       "      <th>published</th>\n",
       "      <th>covid_related</th>\n",
       "      <th>virus</th>\n",
       "      <th>coronavirus</th>\n",
       "      <th>sars</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.</td>\n",
       "      <td>2014-01-10</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.</td>\n",
       "      <td>2014-06-04</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.</td>\n",
       "      <td>2014-07-03</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix</td>\n",
       "      <td>2014-07-25</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.</td>\n",
       "      <td>2014-11-11</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45769</th>\n",
       "      <td>Purpose The recent Ebola epidemic that devastated West Africa has infected and killed more health care providers than any other outbreak in the history of this virus. An improved understanding of pathogen transmission and the institution of strategies to protect health care providers against infection are needed in infectious disease outbreaks. This review connects what is known about Ebola virus transmission with personal protective equipment (PPE) designed to arrest nosocomial transmission. Methods Articles pertaining to filovirus transmission and PPE in filovirus outbreaks were reviewed and findings are presented. In addition, studies that evaluated PPE and donning and doffing strategies are presented. Findings PPE is one step in a comprehensive infection prevention and control strategy that is required to protect health care providers. Given that the Ebola virus is primarily transmitted through direct contact of mucous membranes and cuts in the skin with infected patients and/or their bodily fluids, it is necessary to cover these potential portals of infection with PPE as part of a structured and instructed donning and doffing procedure. Implications Current recommendations about PPE and the donning and doffing processes are based on anecdotal experience. However, the use of non-human viruses can help provide evidence-based guidelines on both PPE and donning and doffing processes.</td>\n",
       "      <td>2015-11-01</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45770</th>\n",
       "      <td>Background Viral respiratory infection has long been known to influence the occurrence of asthma exacerbations. Over the last 20years much effort has been put into clarifying the role that viral respiratory infections play in the eventual development of asthma. Scope of review In this review we give a general background of the role of viruses in the processes of asthma exacerbation and asthma induction. We review recent additions to the literature in the last 3years with particular focus on clinical and epidemiologic investigations of influenza, rhinovirus, bocavirus, respiratory syncytial virus, and metapneumovirus. Major conclusions The development of asthma emerges from a complex interaction of genetic predisposition and environmental factors with viral infection likely playing a significant role in the effect of environment on asthma inception. This article is part of a Special Issue entitled: Biochemistry of Asthma. General significance Further understanding of the role that viruses play in asthma exacerbation and inception will contribute to decreased asthma morbidity in the future. This article is part of a Special Issue entitled: Biochemistry of Asthma.</td>\n",
       "      <td>2011-11-30</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45771</th>\n",
       "      <td>There are no criteria for a pandemic, but covid-19 looks like one, says Debora MacKenzie</td>\n",
       "      <td>2020-03-07</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45772</th>\n",
       "      <td>Communication, transparency key as Canada faces new coronavirus threat</td>\n",
       "      <td>2020-02-17</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45773</th>\n",
       "      <td>Emerging and re-emerging pathogens represent a substantial threat to public health, as demonstrated with numerous outbreaks over the past years, including the 2013–2016 outbreak of Ebola virus in western Africa. Coronaviruses are also a threat for humans, as evidenced in 2002/2003 with infection by the severe acute respiratory syndrome coronavirus (SARS-CoV), which caused more than 8000 human infections with 10% fatality rate in 37 countries. Ten years later, a novel human coronavirus (Middle East respiratory syndrome coronavirus, MERS-CoV), associated with severe pneumonia, arose in the Kingdom of Saudi Arabia. Until December 2016, MERS has accounted for more than 1800 cases and 35% fatality rate. Finding an animal model of disease is key to develop vaccines or antivirals against such emerging pathogens and to understand its pathogenesis. Knowledge of the potential role of domestic livestock and other animal species in the transmission of pathogens is of importance to understand the epidemiology of the disease. Little is known about MERS-CoV animal host range. In this paper, experimental data on potential hosts for MERS-CoV is reviewed. Advantages and limitations of different animal models are evaluated in relation to viral pathogenesis and transmission studies. Finally, the relevance of potential new target species is discussed.</td>\n",
       "      <td>2017-03-03</td>\n",
       "      <td>False</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>45774 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        abstract  \\\n",
       "0                                                                                                                                                                                                                                                                                                                                                                                          Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.   \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.   \n",
       "2                                                                                                                                                                                                                                                                                              Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.   \n",
       "3      Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix   \n",
       "4               Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.   \n",
       "...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          ...   \n",
       "45769                                                                                                                                                                                            Purpose The recent Ebola epidemic that devastated West Africa has infected and killed more health care providers than any other outbreak in the history of this virus. An improved understanding of pathogen transmission and the institution of strategies to protect health care providers against infection are needed in infectious disease outbreaks. This review connects what is known about Ebola virus transmission with personal protective equipment (PPE) designed to arrest nosocomial transmission. Methods Articles pertaining to filovirus transmission and PPE in filovirus outbreaks were reviewed and findings are presented. In addition, studies that evaluated PPE and donning and doffing strategies are presented. Findings PPE is one step in a comprehensive infection prevention and control strategy that is required to protect health care providers. Given that the Ebola virus is primarily transmitted through direct contact of mucous membranes and cuts in the skin with infected patients and/or their bodily fluids, it is necessary to cover these potential portals of infection with PPE as part of a structured and instructed donning and doffing procedure. Implications Current recommendations about PPE and the donning and doffing processes are based on anecdotal experience. However, the use of non-human viruses can help provide evidence-based guidelines on both PPE and donning and doffing processes.   \n",
       "45770                                                                                                                                                                                                                                                                                                                                                                                                                                Background Viral respiratory infection has long been known to influence the occurrence of asthma exacerbations. Over the last 20years much effort has been put into clarifying the role that viral respiratory infections play in the eventual development of asthma. Scope of review In this review we give a general background of the role of viruses in the processes of asthma exacerbation and asthma induction. We review recent additions to the literature in the last 3years with particular focus on clinical and epidemiologic investigations of influenza, rhinovirus, bocavirus, respiratory syncytial virus, and metapneumovirus. Major conclusions The development of asthma emerges from a complex interaction of genetic predisposition and environmental factors with viral infection likely playing a significant role in the effect of environment on asthma inception. This article is part of a Special Issue entitled: Biochemistry of Asthma. General significance Further understanding of the role that viruses play in asthma exacerbation and inception will contribute to decreased asthma morbidity in the future. This article is part of a Special Issue entitled: Biochemistry of Asthma.   \n",
       "45771                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   There are no criteria for a pandemic, but covid-19 looks like one, says Debora MacKenzie   \n",
       "45772                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     Communication, transparency key as Canada faces new coronavirus threat   \n",
       "45773                                                                                                                                                                                                                                                   Emerging and re-emerging pathogens represent a substantial threat to public health, as demonstrated with numerous outbreaks over the past years, including the 2013–2016 outbreak of Ebola virus in western Africa. Coronaviruses are also a threat for humans, as evidenced in 2002/2003 with infection by the severe acute respiratory syndrome coronavirus (SARS-CoV), which caused more than 8000 human infections with 10% fatality rate in 37 countries. Ten years later, a novel human coronavirus (Middle East respiratory syndrome coronavirus, MERS-CoV), associated with severe pneumonia, arose in the Kingdom of Saudi Arabia. Until December 2016, MERS has accounted for more than 1800 cases and 35% fatality rate. Finding an animal model of disease is key to develop vaccines or antivirals against such emerging pathogens and to understand its pathogenesis. Knowledge of the potential role of domestic livestock and other animal species in the transmission of pathogens is of importance to understand the epidemiology of the disease. Little is known about MERS-CoV animal host range. In this paper, experimental data on potential hosts for MERS-CoV is reviewed. Advantages and limitations of different animal models are evaluated in relation to viral pathogenesis and transmission studies. Finally, the relevance of potential new target species is discussed.   \n",
       "\n",
       "       published  covid_related  virus  coronavirus   sars  \n",
       "0     2014-01-10          False  False        False  False  \n",
       "1     2014-06-04          False  False        False   True  \n",
       "2     2014-07-03          False  False        False  False  \n",
       "3     2014-07-25          False   True        False  False  \n",
       "4     2014-11-11          False   True        False  False  \n",
       "...          ...            ...    ...          ...    ...  \n",
       "45769 2015-11-01          False   True        False  False  \n",
       "45770 2011-11-30          False   True        False  False  \n",
       "45771 2020-03-07           True  False        False  False  \n",
       "45772 2020-02-17          False   True         True  False  \n",
       "45773 2017-03-03          False   True         True   True  \n",
       "\n",
       "[45774 rows x 6 columns]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "COLS = ['abstract', 'published', 'covid_related', 'virus', 'coronavirus', 'sars']\n",
    "research_papers = ResearchPapers.load()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<h3>CORD 19 Research Papers</h3>\n",
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Papers</th>\n",
       "      <th>Oldest</th>\n",
       "      <th>Newest</th>\n",
       "      <th>SARS-COV-2</th>\n",
       "      <th>SARS</th>\n",
       "      <th>Coronavirus</th>\n",
       "      <th>Virus</th>\n",
       "      <th>Antivirals</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <td>22760</td>\n",
       "      <td>2002-11-01</td>\n",
       "      <td>2020-12-31</td>\n",
       "      <td>1745</td>\n",
       "      <td>2395</td>\n",
       "      <td>7590</td>\n",
       "      <td>22760</td>\n",
       "      <td>341</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>\n",
       "<hr/>\n",
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>journal</th>\n",
       "      <th>authors</th>\n",
       "      <th>published</th>\n",
       "      <th>when</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Bayesian mixture analysis for metagenomic community profiling.</td>\n",
       "      <td>Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix</td>\n",
       "      <td></td>\n",
       "      <td>Sofia Morfopoulou; Vincent Plagnol</td>\n",
       "      <td>2014-07-25</td>\n",
       "      <td>6 years ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Mapping a viral phylogeny onto outbreak trees to improve host transmission inference</td>\n",
       "      <td>Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.</td>\n",
       "      <td></td>\n",
       "      <td>Stephen P Velsko; Jonathan E Allen</td>\n",
       "      <td>2014-11-11</td>\n",
       "      <td>5 years ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>The infant airway microbiome in health and disease impacts later asthma development</td>\n",
       "      <td>The nasopharynx (NP) is a reservoir for microbes associated with acute respiratory illnesses (ARI). The development of asthma is initiated during infancy, driven by airway inflammation associated with infections. Here, we report viral and bacterial community profiling of NP aspirates across a birth cohort, capturing all lower respiratory illnesses during their first year. Most infants were initially colonized with Staphylococcus or Corynebacterium before stable colonization with Alloiococcus or Moraxella, with transient incursions of Streptococcus, Moraxella or Haemophilus marking virus-associated ARIs. Our data identify the NP microbiome as a determinant for infection spread to the lower airways, severity of accompanying inflammatory symptoms, and risk for future asthma development. Early asymptomatic colonization with Streptococcus was a strong asthma predictor, and antibiotic usage disrupted asymptomatic colonization patterns.</td>\n",
       "      <td></td>\n",
       "      <td>Shu Mei Teo; Danny Mok; Kym Pham; Merci Kusel; Michael Serralha; Niamh Troy; Barbara J Holt; Belinda J Hales; Michael L Walker; Elysia Hollams; Yury H Bochkov; Kristine Grindle; Sebastian L Johnston; James E Gern; Peter D Sly; Patrick G Holt; Kathryn E Holt; Michael Inouye</td>\n",
       "      <td>2014-12-02</td>\n",
       "      <td>5 years ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Big city, small world: Density, contact rates, and transmission of dengue across Pakistan.</td>\n",
       "      <td>Macroscopic descriptions of populations commonly assume that encounters between individuals are well mixed; i.e., each individual has an equal chance of coming into contact with any other individual. Relaxing this assumption can be challenging though, due to the difficulty of acquiring detailed knowledge about the non-random nature of encounters. Here, we fitted a mathematical model of dengue virus transmission to spatial time series data from Pakistan and compared maximum-likelihood estimates of “mixing parameters” when disaggregating data across an urban-rural gradient. We show that dynamics across this gradient are subject not only to differing transmission intensities but also to differing strengths of nonlinearity due to differences in mixing. We furthermore show that neglecting spatial variation in mixing can lead to substantial underestimates of the level of effort needed to control a pathogen with vaccines or other control efforts. We complement this analysis with relevant contemporary environmental drivers of dengue.</td>\n",
       "      <td></td>\n",
       "      <td>Moritz U. G. Kraemer; T. Alex Perkins; Derek A.T. Cummings; Rubeena Zakar; Simon I. Hay; David L. Smith; Robert C. Reiner</td>\n",
       "      <td>2015-04-27</td>\n",
       "      <td>5 years ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>MERS-CoV recombination: implications about the reservoir and potential for adaptation</td>\n",
       "      <td>Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology.</td>\n",
       "      <td></td>\n",
       "      <td>Gytis Dudas; Andrew Rambaut</td>\n",
       "      <td>2015-06-12</td>\n",
       "      <td>5 years ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45766</th>\n",
       "      <td>Modulation of influenza virus replication by alteration of sodium ion transport and protein kinase C activity</td>\n",
       "      <td>In recent years, increasing levels of resistance to the four FDA-approved anti-influenza virus drugs have been described and vaccine manufacturers have experienced demands that exceed their capacity. This situation underlines the urgent need for novel antivirals as well as innovations in vaccine production in preparation for the next influenza epidemic. Here we report the development of a cell-based high-throughput screen which we have used for the identification of compounds that modulate influenza virus growth either negatively or positively. We screened a library of compounds with known biological activity and identified distinct groups of inhibitors and enhancers that target sodium channels or protein kinase C (PKC). We confirmed these results in viral growth assays and find that treatment with a sodium channel opener or PKC inhibitor significantly reduces viral replication. In contrast, inhibition of sodium channels or activation of PKC leads to enhanced virus production in tissue culture. These diametrically opposing effects strongly support a role for PKC activity and the regulation of Na+ currents in influenza virus replication and both may serve as targets for antiviral drugs. Furthermore, we raise the possibility that compounds that result in increased viral titers may be beneficial for boosting the production of tissue culture-grown influenza vaccines.</td>\n",
       "      <td>Antiviral Research</td>\n",
       "      <td>Hoffmann, H.-Heinrich; Palese, Peter; Shaw, Megan L.</td>\n",
       "      <td>2008-11-30</td>\n",
       "      <td>11 years ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45769</th>\n",
       "      <td>Personal Protective Equipment: Protecting Health Care Providers in an Ebola Outbreak</td>\n",
       "      <td>Purpose The recent Ebola epidemic that devastated West Africa has infected and killed more health care providers than any other outbreak in the history of this virus. An improved understanding of pathogen transmission and the institution of strategies to protect health care providers against infection are needed in infectious disease outbreaks. This review connects what is known about Ebola virus transmission with personal protective equipment (PPE) designed to arrest nosocomial transmission. Methods Articles pertaining to filovirus transmission and PPE in filovirus outbreaks were reviewed and findings are presented. In addition, studies that evaluated PPE and donning and doffing strategies are presented. Findings PPE is one step in a comprehensive infection prevention and control strategy that is required to protect health care providers. Given that the Ebola virus is primarily transmitted through direct contact of mucous membranes and cuts in the skin with infected patients and/or their bodily fluids, it is necessary to cover these potential portals of infection with PPE as part of a structured and instructed donning and doffing procedure. Implications Current recommendations about PPE and the donning and doffing processes are based on anecdotal experience. However, the use of non-human viruses can help provide evidence-based guidelines on both PPE and donning and doffing processes.</td>\n",
       "      <td>Clinical Therapeutics</td>\n",
       "      <td>Fischer, William A.; Weber, David J.; Wohl, David A.</td>\n",
       "      <td>2015-11-01</td>\n",
       "      <td>4 years ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45770</th>\n",
       "      <td>Viruses and asthma</td>\n",
       "      <td>Background Viral respiratory infection has long been known to influence the occurrence of asthma exacerbations. Over the last 20years much effort has been put into clarifying the role that viral respiratory infections play in the eventual development of asthma. Scope of review In this review we give a general background of the role of viruses in the processes of asthma exacerbation and asthma induction. We review recent additions to the literature in the last 3years with particular focus on clinical and epidemiologic investigations of influenza, rhinovirus, bocavirus, respiratory syncytial virus, and metapneumovirus. Major conclusions The development of asthma emerges from a complex interaction of genetic predisposition and environmental factors with viral infection likely playing a significant role in the effect of environment on asthma inception. This article is part of a Special Issue entitled: Biochemistry of Asthma. General significance Further understanding of the role that viruses play in asthma exacerbation and inception will contribute to decreased asthma morbidity in the future. This article is part of a Special Issue entitled: Biochemistry of Asthma.</td>\n",
       "      <td>Biochimica et Biophysica Acta (BBA) - General Subjects</td>\n",
       "      <td>Dulek, Daniel E.; Peebles, R. Stokes</td>\n",
       "      <td>2011-11-30</td>\n",
       "      <td>8 years ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45772</th>\n",
       "      <td>Communication, transparency key as Canada faces new coronavirus threat</td>\n",
       "      <td>Communication, transparency key as Canada faces new coronavirus threat</td>\n",
       "      <td>Canadian Medical Association Journal</td>\n",
       "      <td>Glauser, Wendy</td>\n",
       "      <td>2020-02-17</td>\n",
       "      <td>1 month ago</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45773</th>\n",
       "      <td>Searching for animal models and potential target species for emerging pathogens: Experience gained from Middle East respiratory syndrome (MERS) coronavirus</td>\n",
       "      <td>Emerging and re-emerging pathogens represent a substantial threat to public health, as demonstrated with numerous outbreaks over the past years, including the 2013–2016 outbreak of Ebola virus in western Africa. Coronaviruses are also a threat for humans, as evidenced in 2002/2003 with infection by the severe acute respiratory syndrome coronavirus (SARS-CoV), which caused more than 8000 human infections with 10% fatality rate in 37 countries. Ten years later, a novel human coronavirus (Middle East respiratory syndrome coronavirus, MERS-CoV), associated with severe pneumonia, arose in the Kingdom of Saudi Arabia. Until December 2016, MERS has accounted for more than 1800 cases and 35% fatality rate. Finding an animal model of disease is key to develop vaccines or antivirals against such emerging pathogens and to understand its pathogenesis. Knowledge of the potential role of domestic livestock and other animal species in the transmission of pathogens is of importance to understand the epidemiology of the disease. Little is known about MERS-CoV animal host range. In this paper, experimental data on potential hosts for MERS-CoV is reviewed. Advantages and limitations of different animal models are evaluated in relation to viral pathogenesis and transmission studies. Finally, the relevance of potential new target species is discussed.</td>\n",
       "      <td>One Health</td>\n",
       "      <td>Vergara-Alert, Júlia; Vidal, Enric; Bensaid, Albert; Segalés, Joaquim</td>\n",
       "      <td>2017-03-03</td>\n",
       "      <td>3 years ago</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>22760 rows × 6 columns</p>\n",
       "</div>\n",
       "<style>\n",
       "h3 {color: #4169E1}\n",
       "</style>"
      ],
      "text/plain": [
       "<cord.cord19.ResearchPapers at 0x26608e67470>"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "research_papers.since_sars().query('virus')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "cord",
   "language": "python",
   "name": "cord"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
